src/java.base/share/classes/java/util/concurrent/ForkJoinPool.java

```diff
@@ -855,7 +855,7 @@ public class ForkJoinPool extends AbstractExecutorService {
      * non-workers (which comply with Future.get() specs). Internal
      * usages of ForkJoinTasks ignore interrupt status when executing
      * or awaiting completion.  Otherwise, reporting task results or
-     * exceptions is preferred to throwing InterruptedExecptions,
+     * exceptions is preferred to throwing InterruptedExceptions,
```

Comment:
I wonder whether _“InterruptedExceptions”_ should be marked up with `{@code InterruptedException}`s to refer to the class. As far as I can see, classes and methods aren't marked up with `{@code}` here, so it's better to leave it as is.

Reply:
That comment is not javadoc, so no need for markup. Although, I agree that this trailing "s" reads ugly because it interferes with the class name.

Reply:
It would also be OK to just drop that "s" -> InterruptedException.

Reply:
> It would also be OK to just drop that "s" -> InterruptedException.

True, but there are other similar cases in that same file, maybe it's okay to leave it like that. Similar cases: ForkJoinTasks, Futures, CountedCompleters, InterruptibleTasks, WorkQueues, Deques, Workers, ForkJoinPools, SecurityManagers, ThreadLocals, VirtualThreads, ForkJoinWorkerThreads, VarHandles (this one actually clashes with an existing class).